test/compression.js

```diff
@@ -605,9 +588,7 @@ describe('compression()', function () {
         .expect('Content-Encoding', 'br', done)
     })
 
-    brotli = hasBrotliSupport ? it.skip : it
-
-    brotli('should respond with gzip', function (done) {
+    it.skip('should respond with gzip', function (done) {
```

Comment:
I set this test to `it.skip` because we may need it since there is an open issue for it: #220